jpos/src/main/java/org/jpos/space/JESpace.java

```diff
@@ -71,14 +71,20 @@
     static final Map<String,Space> spaceRegistrar = 
         new HashMap<String,Space> ();
 
-    public JESpace(String name, String params) throws SpaceError {
+    public JESpace(String name, String params) {
+        this(name, params, true);
+    }
+
+    public JESpace(String name, String params, boolean sync) throws SpaceError {
         super();
         try {
             EnvironmentConfig envConfig = new EnvironmentConfig();
             StoreConfig storeConfig = new StoreConfig();
             String[] p = ISOUtil.commaDecode(params);
             String path = p[0];
             envConfig.setAllowCreate (true);
+            if (!sync)
```

Comment:
See my comment in JDBMSpace. Having around an easy to misconfigure JESpace due to no syncs is a recipe for disaster.